src/org/jitsi/impl/neomedia/codec/video/vp8/DePacketizer.java

```diff
@@ -507,8 +507,30 @@ public static int getTemporalLayerIndex(byte[] buf, int off, int len)
          * The size in bytes of the Payload Descriptor at offset
          * <tt>offset</tt> in <tt>input</tt>. The size is between 1 and 6.
          *
+         * @param baf the <tt>ByteArrayBuffer</tt> that holds the VP8 payload
+         * descriptor.
+         *
+         * @return The size in bytes of the Payload Descriptor at offset
+         * <tt>offset</tt> in <tt>input</tt>, or -1 if the input is not a valid
+         * VP8 Payload Descriptor. The size is between 1 and 6.
+         */
+        public static int getSize(ByteArrayBuffer baf)
```

Comment:
This is merely adding an overloading an existing getSize() method, I'd rather not change the architecture + it's specific to VPX payload descriptor packetization.